locations/spiders/DskBank.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+import scrapy
+from scrapy import Selector
+
+from locations.categories import Categories, apply_category
+from locations.dict_parser import DictParser
+from locations.hours import DAYS_BG, OpeningHours
+
+
+class DSKBankBGSpider(scrapy.Spider):
+    name = "dsk_bank_bg"
+    item_attributes = {"brand": "Банка ДСК", "brand_wikidata": "Q5206146"}
+    allowed_domains = ["https://www.dskbank.bg/"]
+    start_urls = ["https://dskbank.bg/контакти/клонова-мрежа/GetOffices/"]
+
+    def parse(self, response):
+        for data in response.json():
+            item = DictParser.parse(data)
+
+            branch_type = data["BranchType"]
+            if "Branch" in branch_type:
+                apply_category(Categories.BANK, item)
+            else:
+                apply_category(Categories.ATM, item)
+
+            if data["Phone"]:
+                item["phone"] = data["Phone"].split(";", 1)[0]
+            else:
+                item["phone"] = None
```

Comment:
alright!

Reply:
> Alright! But this should raise an `Attribute error` because when `data["Phone"]` is None the `spilt` function cannot be applied to it.

The second example `item["phone"] = data.get("Phone", "").split(";", 1)[0]` would need to be used then if data["Phone"] is sometimes `None`. Basically `item["phone"] = ""` (empty string) and a Scrapy middleware will ignore the field because it isn't a valid phone number.